project/frontend/src/FindMatch/FindMatch.js

```diff
@@ -6,35 +6,29 @@ class PreferenceForm extends React.Component {
     super(props)
   }
   render() {
-    const cuisines = ["Italian", "Mexican", "Indian"]
+    const cuisines = ["Italian", "Mexican", "Indian"];
+    const distances = ["1 mile", "5 miles", "10 miles", "25 miles"];
+    const dining_experiences = ["Takeout", "Delivery", "Dine-In"];
```

Comment:
Change to `const distances = [1, 5, 10, 25]` and `const dining_experiences = ['meal_takeaway', 'meal_delivery', 'restaurant']`. This will make it easier to use with the API :)